src/Annotation/Controller/RequestParameter.php

```diff
@@ -34,36 +36,31 @@ final class RequestParameter
 	/** @var bool */
 	private $allowEmpty;
 
-	/**
-	 * @param mixed[] $values
-	 */
-	public function __construct(array $values)
+	public function __construct(
+		string $name,
+		string $type,
+		bool $required = true,
+		bool $allowEmpty = false,
+		bool $deprecated = false,
+		?string $description = null,
+		string $in = EndpointParameter::IN_PATH
```

Comment:
`in` should be before required. It's more important.

Reply:
Done :+1: